<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusyProjectManager::Internal::BusyBuildStepConfigWidget</class>
 <widget class="QWidget" name="BusyProjectManager::Internal::BusyBuildStepConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>247</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="buildVariantLabel">
     <property name="text">
      <string>Build variant:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QComboBox" name="buildVariantComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Debug</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Release</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>70</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="jobsLabel">
     <property name="text">
      <string>Parallel jobs:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QSpinBox" name="jobSpinBox">
       <property name="toolTip">
        <string>Number of concurrent build jobs.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="targetsLabel">
     <property name="text">
      <string>Targets:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="Utils::FancyLineEdit" name="targetsEdit">
     <property name="toolTip">
      <string>Properties to pass to the project.</string>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="flagsLabel">
     <property name="text">
      <string>Flags:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="stopOnError">
       <property name="toolTip">
        <string>Stop the build on the first compiler or linker run with errors.</string>
       </property>
       <property name="text">
        <string>Stop build on error</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="trackHeaders">
       <property name="toolTip">
        <string>When compiling a source file also check the modification date of the headers it depends on to decide whether compilation is required or not.</string>
       </property>
       <property name="text">
        <string>Track header dependencies</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showCommandLinesCheckBox">
       <property name="text">
        <string>Show command lines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="installCheckBox">
       <property name="text">
        <string>Install</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cleanInstallRootCheckBox">
       <property name="text">
        <string>Clean install root</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="checkBoxSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="parametersKeyLabel">
     <property name="text">
      <string>Parameters:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPlainTextEdit" name="parametersTextEdit">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="plainText">
      <string notr="true"/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextEditorInteraction</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buildVariantLabel</zorder>
  <zorder>targetsLabel</zorder>
  <zorder>targetsEdit</zorder>
  <zorder>flagsLabel</zorder>
  <zorder>jobsLabel</zorder>
  <zorder>parametersTextEdit</zorder>
  <zorder>parametersKeyLabel</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/fancylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../core/core.qrc"/>
 </resources>
 <connections/>
</ui>
